src/components/data-table-v2/data-table-v2.js

```diff
@@ -89,6 +89,14 @@ class DataTableV2 extends mixin(createComponent, initComponentBySearch, eventedS
     this.state.checkboxCount += checked ? 1 : -1;
     this.countEl.textContent = this.state.checkboxCount;
 
+    const row = element.parentNode.parentNode;
+
+    if (checked && row) {
+      row.classList.add('bx--data-table-v2--selected');
```

Comment:
Could we just `row.classList.toggle` instead of using the if/else conds?

Reply:
Yeah we could shorten it to that, updating

Reply:
Actually, will need to keep it in the select-all check logic, because if you click one row, then check all, the one selected is toggled off